src/components/containers/navbar.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import CommonNavItems from '../views/commonNavItems';
+import UserNavItems from '../views/userNavItems';
+import GuestNavItems from '../views/guestNavItems';
+
+
+const NavBar = ({ isLoggedIn }) => {
+  return (
+    <div className="layout">
+
+      <nav className="navbar general__nav">
+        <div className="container">
+          <div className="navbar-brand">
+            <a className="navbar-item" href="#">
```

Comment:
React-Router `Link` can be used here in place of `a` tags since we have the react-router already.

Reply:
Nice catch. Thanks Theo